src/ProjectOrigin.WalletSystem.IntegrationTests/QueryCertificatesTest.cs

```diff
@@ -56,37 +56,111 @@ public async void QueryCertificates()
             using (var connection = new NpgsqlConnection(_dbFixture.ConnectionString))
             {
                 var walletRepository = new WalletRepository(connection);
-                var wallet = new Wallet(Guid.NewGuid(), owner, Algorithm.GenerateNewPrivateKey());
-                var notOwnedWallet = new Wallet(Guid.NewGuid(), someOtherOwner, Algorithm.GenerateNewPrivateKey());
+                var wallet = new Wallet
+                {
+                    Id = Guid.NewGuid(),
+                    Owner = owner,
+                    PrivateKey = Algorithm.GenerateNewPrivateKey()
+                };
+                var notOwnedWallet = new Wallet
+                {
+                    Id = Guid.NewGuid(),
+                    Owner = someOtherOwner,
+                    PrivateKey = Algorithm.GenerateNewPrivateKey()
+                };
                 await walletRepository.Create(wallet);
                 await walletRepository.Create(notOwnedWallet);
 
                 var depositEndpoint = await walletRepository.CreateDepositEndpoint(wallet.Id, string.Empty);
                 var notOwnedDepositEndpoint = await walletRepository.CreateDepositEndpoint(notOwnedWallet.Id, string.Empty);
 
                 var regName = _fixture.Create<string>();
-                var registry = new RegistryModel(Guid.NewGuid(), regName);
                 var certificateRepository = new CertificateRepository(connection);
-                var registryRepository = new RegistryRepository(connection);
-                await registryRepository.InsertRegistry(registry);
 
                 var attributes = new List<CertificateAttribute>
                 {
-                    new ("AssetId", "571234567890123456"),
-                    new ("TechCode", "T070000"),
-                    new ("FuelCode", "F00000000")
+                    new (){Key="AssetId", Value= "571234567890123456"},
+                    new (){Key="TechCode", Value= "T070000"},
+                    new (){Key="FuelCode", Value= "F00000000"}
```

Comment:
Formatting of these lines is inconsistent.

Reply:
fixed